intranet/apps/welcome/views.py

```diff
@@ -16,7 +18,10 @@ def student_welcome_view(request):
 
 @login_required
 def teacher_welcome_view(request):
-    context = {"first_login": request.session["first_login"] if "first_login" in request.session else False}
+    context = {
+        "first_login": request.session["first_login"] if "first_login" in request.session else False,
+        "coordinator": settings.EIGHTH_COORDINATOR_NAME,
```

Comment:
Since d277b79a8add567026888cd10ca20219965d921b, we export the `settings` module as the context variable `DJANGO_SETTINGS`. Perhaps we should use that instead of adding it to the context here?